src/easypost.js

```diff
@@ -123,10 +126,16 @@ export default class API {
     } = params;
 
     const req = this.agent[method](this.buildPath(path))
-                  .accept('json')
-                  .set('Content-Type', 'application/json')
-                  .set(API.buildHeaders(headers))
-                  .auth(`${this.key}:`);
+                .accept('json')
+                .set('Content-Type', 'application/json')
+                .set(API.buildHeaders(headers))
+                .auth(`${this.key}:`);
+
+    if (path === 'rating/v1/rates') {
```

Comment:
> also the base url is https://api.easypost.com/v2 , but the url required for rating is just https://api.easypost.com/rating/v1/rates

Let's change the base to `https://api.easypost.com` then, and update the endpoints to include the `v2` prefix. The base class shouldn't have anything specific to individual endpoints - all the code for each endpoint should be contained entirely within its own class.